src/main/java/com/iota/iri/service/milestone/impl/MilestoneSolidifierImpl.java

```diff
@@ -30,12 +30,12 @@
     /**
      * Defines the amount of milestones that we "simultaneously" try to solidify in one pass.
      */
-    private static final int SOLIDIFICATION_QUEUE_SIZE = 10;
+    private static final int SOLIDIFICATION_QUEUE_SIZE = 20;
 
     /**
      * Defines the interval in which solidity checks are issued (in milliseconds).
      */
-    private static final int SOLIDIFICATION_INTERVAL = 100;
+    private static final int SOLIDIFICATION_INTERVAL = 50;
```

Comment:
Can you just change in the description: 
"*Increases* the milestone solidifier interval to 50 milliseconds" -> "*Decreases* the milestone solidifier interval to 50 milliseconds"

Or you can write that you increase the rate, if you think "increase" sounds more positive...

I just got confused and so will others :-P

Reply:
Changed